.circleci/config.yml

```diff
@@ -70,8 +82,6 @@ jobs:
               sudo pip install rethinkdb==2.3.0
               rethinkdb-restore --force packages/cypress/fixtures/rdb_test.tar.gz
               node scripts/toolbox/softenDurability.js
-              yarn db:migrate
```

Comment:
i haven't tested this, but note that these are called after 
`cp $DEVOPS_WORKDIR/environments/cypress .env`
if we execute these commands before copying over the cypress env then we are executing them on a completely different set of environment variables, which probably isn't what you want.

@tiffanyhan worth a hotfix before going to prod?

Reply:
shoot thank you for looking into this @mattkrick. i really misinterpreted the sequence of steps here. i'll link you to the hotfix.